### PR TITLE
 Improve the performance of <org.apache.skywalking.oap.server.library.module.ModuleManager#init>

### PR DESCRIPTION
**Pull request by panniyuyu**
- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [ ] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

### CODE REVIEW

File: oap-server/server-library/library-module/src/main/java/org/apache/skywalking/oap/server/library/module/ModuleManager.java [hunk 1]
```diff
@@ -40,21 +40,19 @@ public void init(
...
+        HashSet<String> moduleSet = new HashSet<>(Arrays.asList(moduleNames));
         for (ModuleDefine module : moduleServiceLoader) {
-            for (String moduleName : moduleNames) {
-                if (moduleName.equals(module.name())) {
-                    module.prepare(this, applicationConfiguration.getModuleConfiguration(moduleName), moduleProviderLoader);
-                    loadedModules.put(moduleName, module);
-                    moduleList.remove(moduleName);
-                }
+            if (moduleSet.contains(module.name())) {
+                module.prepare(this, applicationConfiguration.getModuleConfiguration(module.name()), moduleProviderLoader);
```
[the remark this answers is not in the document]
Comment (panniyuyu) [reply oap-server/server-library/library-module/src/main/java/org/apache/skywalking/oap/server/library/module/ModuleManager.java:46]: ignore

[the remark this answers is not in the document]
Comment (panniyuyu) [reply oap-server/server-library/library-module/src/main/java/org/apache/skywalking/oap/server/library/module/ModuleManager.java:46]: @muse-dev ignore

**Review by wu-sheng — APPROVED** [review]
The performance here doesn't matter but also doesn't have harm to improve.

**Comment by panniyuyu** [comment]
> The performance here doesn't matter but also doesn't have harm to improve.

This should be a little better.As a junior programmer, I will continue to study in depth. Thanks for merging

**Pull request merged**